### PR TITLE
added instructions on where to find Developer:

### PR DESCRIPTION
**Pull request by drscotthawley**
this can close Issue https://github.com/microsoft/vscode/issues/134599.

### CODE REVIEW

File: Performance-Issues.md [hunk 1]
```diff
@@ -46,15 +46,15 @@ If you cannot reproduce the issue when **all** extensions are disabled, then can
...
 
 <a name="profile-running-extensions"/>
 
 ### Profile the Running Extensions
 
 You can create a CPU profile and share it in the issue with the extension author or us. To create a CPU profile:
 - Close all instances of VSCode and start with `code --inspect-extensions=9993` or any other port number.
-- Execute the `Developer: Show Running Extensions` Command. This command opens an editor with all the running extensions.
+- Execute the `Developer: Show Running Extensions` Command (found via View > Command Pallette... > Developer: Running Extensions). This command opens an editor with all the running extensions.
 - To start recording a profile click the run control in the editor's title bar:
```
Comment (jrieken) [inline Performance-Issues.md:58]: Maybe align with L166, e.g "Select "F1 > Developer: Show Running Extensions", This command...".

**Review by jrieken — CHANGES_REQUESTED** [review]
suggest to tweak wording towards the fast path

File: Performance-Issues.md (hunk 1 above)
Comment (drscotthawley) [reply Performance-Issues.md:58]: Done!

**Review by jrieken — APPROVED** [review]
LGTM. Thanks

**Pull request merged**